packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -15,10 +15,13 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper-operator/charts-ori
    - https://github.com/open-policy-agent/gatekeeper.git
  appVersion: v3.1.0-beta.7
 +annotations:
-+  io.rancher.certified: experimental
-diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper-operator/charts-original/helm-modifications/helm-modifications.yaml packages/rancher-gatekeeper-operator/charts/helm-modifications/helm-modifications.yaml
---- packages/rancher-gatekeeper-operator/charts-original/helm-modifications/helm-modifications.yaml
-+++ packages/rancher-gatekeeper-operator/charts/helm-modifications/helm-modifications.yaml
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/experimental: true
++  catalog.rancher.io/namespace: gatekeeper-system
++  catalog.rancher.io/release-name: opa
```

Comment:
Can we rename this to `rancher-gatekeeper`?

Reply:
Done